transmart-solr-indexing/src/main/groovy/org/transmartproject/solr/modules/DataTypesIndexingModule.groovy

```diff
@@ -8,14 +8,17 @@ import org.transmartproject.core.dataquery.highdim.HighDimensionResource
 import org.transmartproject.core.dataquery.highdim.assayconstraints.AssayConstraint
 import org.transmartproject.core.ontology.StudiesResource
 import org.transmartproject.core.ontology.Study
-import org.transmartproject.search.browse.FolderStudyMappingView
-import org.transmartproject.search.indexing.*
-
-import static org.transmartproject.search.indexing.FacetsFieldImpl.create as createFF
-import static org.transmartproject.search.indexing.FacetsIndexingService.FIELD_CONCEPT_PATH
-import static org.transmartproject.search.indexing.FacetsIndexingService.FIELD_FOLDER_ID
-import static org.transmartproject.search.indexing.modules.AbstractFacetsIndexingFolderModule.FOLDER_DOC_TYPE
-import static org.transmartproject.search.indexing.modules.ConceptNamesIndexingModule.CONCEPT_DOC_TYPE
+import org.transmartproject.solr.FacetsDocId
+import org.transmartproject.solr.FacetsDocument
+import org.transmartproject.solr.FacetsFieldDisplaySettings
+import org.transmartproject.solr.FacetsFieldImpl
+import org.transmartproject.solr.FacetsIndexingModule
+import org.transmartproject.solr.FolderStudyMappingView
+
+import static FacetsFieldImpl.create as createFF
```

Comment:
It's called Static import aliasing. Can be used for things like:
import static Calendar.getInstance as now